aws/signer/v4/middleware_test.go

```diff
@@ -108,6 +112,17 @@ func TestSignHTTPRequestMiddleware(t *testing.T) {
 		"nil creds": {
 			creds: nil,
 		},
+		"with logger": {
+			creds:  unit.StubCredentialsProvider{},
+			hash:   "0123456789abcdef",
+			logger: logging.NewStandardLogger(os.Stdout),
+		},
+		"with log signing": {
+			creds:      unit.StubCredentialsProvider{},
+			hash:       "0123456789abcdef",
+			logger:     logging.NewStandardLogger(os.Stdout),
```

Comment:
How does this asset that it correctly logged signed requests?

Reply:
yeah this doesn't assert anything particular is logged since the Signer is mocked out there is no actual behavior that is being checked. 

Can update the logic so that the signer does use the logger if the flag is set, and check some expected content.